fuse_examples/classification/knight/baseline/clinical_processor.py

```diff
@@ -44,16 +43,61 @@ def __init__(self, json_filename: str, columns_to_tensor: Optional[Union[List[st
                         When None (default) no columns are converted.
         """
         self.json_filename = json_filename
-        self.data = create_knight_clinical(json_filename)
+        self.data = pd.read_json(json_filename)
```

Comment:
I missed it before.
Since we might process the sample with multiple processes, it's better to hold the data in dictionary rather than dataframe.
Can you convert it to a dictionary here? (look at FuseProcessorDataframe for reference) - it should improve the running time.

Reply:
done. please check